sw/host/opentitanlib/src/test_utils/load_sram_program.rs

```diff
@@ -124,12 +196,42 @@ pub fn load_and_execute_sram_program(
     load_sram_program(jtag, &vmem, sram_load_addr)?;
     log::info!("Preparing ePMP for execution");
     prepare_epmp_for_sram(jtag)?;
-    setup_stack(jtag, top_stack_addr, stack_size)?;
-    log::info!("set SP to {:x}", top_stack_addr);
-    jtag.write_riscv_reg(&RiscvReg::GprByName(RiscvGpr::SP), top_stack_addr)?;
+    setup_stack(jtag, stack_pointer, stack_size)?;
+    log::info!("set SP to {:x}", stack_pointer);
+    jtag.write_riscv_reg(&RiscvReg::GprByName(RiscvGpr::SP), stack_pointer)?;
     log::info!("set GP to {:x}", global_pointer);
     jtag.write_riscv_reg(&RiscvReg::GprByName(RiscvGpr::GP), global_pointer)?;
-    log::info!("resume execution at {:x}", sram_load_addr);
-    jtag.resume_at(sram_load_addr)?;
-    Ok(())
+    // to avoid unexpected behaviors, we always make sure that the return addreess
```

Comment:
s/addreess/address/

Reply:
ah let me addreess that 😉 in a follow up, since this is blocking development of other tests.